### PR TITLE
Making random filename XML-element safe

### PR DESCRIPTION
**Pull request by cschroed-usgs**

### CODE REVIEW

File: coastal-hazards-wps/src/test/java/gov/usgs/cida/coastalhazards/wps/FetchAndUnzipProcessTest.java [hunk 1]
```diff
@@ -157,7 +161,27 @@ public void testUnzipSingleFileZipToDir() throws FileNotFoundException, IOExcept
         String expectedContents = "test\n";
         assertEquals(expectedContents, actualContents);
     }
-          
+    @Test
+    public void testMakeSafeFileName(){
+        for(int i = 0; i < 1000; i++){
```
Comment (cschroed-usgs) [inline coastal-hazards-wps/src/test/java/gov/usgs/cida/coastalhazards/wps/FetchAndUnzipProcessTest.java]: The implementation relies on randomness at runtime. To provoke more potentially bad responses, we iterate.

Comment (smlarson-usgs) [inline coastal-hazards-wps/src/test/java/gov/usgs/cida/coastalhazards/wps/FetchAndUnzipProcessTest.java]: Small change looks fine. Did you have the other changes reviewed by Jordan?

Comment (cschroed-usgs) [inline coastal-hazards-wps/src/test/java/gov/usgs/cida/coastalhazards/wps/FetchAndUnzipProcessTest.java]: Thanks! I will merge.

Comment (cschroed-usgs) [inline coastal-hazards-wps/src/test/java/gov/usgs/cida/coastalhazards/wps/FetchAndUnzipProcessTest.java]: RE: Jwalker review. Probably, but I'm not sure? Which changes? Sorry, there's a quite a few pull requests to keep track of :)

**Pull request merged**